developer/src/kmc-keyboard-info/src/font-family.ts

```diff
@@ -16,12 +16,7 @@ export async function getFontFamily(source: Uint8Array) {
   }
 
   const buffer = Buffer.from(source);
-  let font = null;
-  try {
-    font = await ttfMeta.promise(buffer);
-  } catch(e) {
-    return null;
-  }
+  const font = await ttfMeta.promise(buffer);
```

Comment:
We need to update the comment on line 9 because an invalid font may now throw an exception.

Reply:
getFontFamily() is only called in one place, from within KeyboardInfoCompiler.fontSourceToKeyboardInfoFont(), where we now catch the excptions (that could always have been thrown).  I have added a @throws comment.